recipes/scmdata/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: netcdf-scm
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a9/71/9ea5101e0a2689d0fb8ddbd6c6db9ad77144292f9cedc5a7d78b9a0faddb/scmdata-{{ version }}.tar.gz
```

Comment:
Please generate the recipe with `conda-skeleton pypi scmdata`, and modify it afterwards.

Reply:
thanks!